CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 1.9.2 (unreleased)
 ==================
 
+cube_build
```

Comment:
This entry needs to be moved up into the new 1.9.3 (unreleased) section of the log.